### DISCUSSION TITLE
How To Increase Timeouts for Large Volume Snapshots

### DISCUSSION THREAD

**Discussion by btzq** [apache/cloudstack#9556] [Q&A] [answered]
Hi Community, 

We have a scenario where we would backup a large volume (2-5TB) using Volume Snapshot. 

This results in the operation taking more than 1 hour.

However, I think there is a Timeout somewhere cause we notice that any volumes taking more than 1 hour will fail and get timed out. In our case, we use linstor. So what happens is that the volume backup goes missing in Cloudstack GUI, but the volume still exists in Linstor, resulting in lots of orphan volumes (that have not completed the snapshot). 

Is there a specific parameter in Global Settings to tune this? Only for Volume Snapshot scenarios.

**Comment by rajujith** [comment, marked as answer]
@btzq  can you increase backup.snapshot.wait and try?

**Reply by btzq** [reply]
Thanks @rajujith , have changed this parameter to and will give this a try. 

On Average, our Volumes are around 5TB to BackUp. So we have set 36000Minutes in the backup.snapshot.wait. Will reach out again if there is any issues.
